src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.targets

```diff
@@ -1,20 +1,28 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <ItemGroup>
-    <Compile Remove="App_Data\**" />
-    <Content Remove="App_Data\**" />
-    <EmbeddedResource Remove="App_Data\**" />
-    <None Remove="App_Data\**" />
+    <Compile Remove="App_Data\**;Localization\**" />
+    <Content Remove="App_Data\**;Localization\**" />
+    <EmbeddedResource Remove="App_Data\**;Localization\**" />
+    <None Remove="App_Data\**;Localization\**" />
   </ItemGroup>
 
   <ItemGroup>
     <ResolvedFileToPublish Include="App_Data\**" Exclude="App_Data\**\*.log">
       <RelativePath>App_Data\%(RecursiveDir)%(Filename)%(Extension)</RelativePath>
     </ResolvedFileToPublish>
+
+    <ResolvedFileToPublish Include="Localization\**" Exclude="Localization\is-cache\**">
```

Comment:
What is this `Exclude="Localization\is-cache\**`

Reply:
oh sorry i'm too tired, i will fix it